README.md

```diff
@@ -137,7 +132,6 @@ We do NOT embed any i18n data within our library. However, we make it really
 easy to use. Read [How to get and load CLDR JSON data](#cldr) for more
```

Comment:
`#cldr` needs to be updated to `#2-cldr-content`. The same should be done on an occurrence of `doc/api/core/load.md`.